### ISSUE TITLE
How can I get the contribution of each modality?

### ISSUE THREAD

**Issue by antecede** [Justin1904/TensorFusionNetworks#12]
Dear Justin,

I saw the article in `https://towardsdatascience.com/multimodal-deep-learning-ce7d1d994f4`, but I did not see the code about the contribution of each modality. Besides, I saw some descriptions about the loss function on the website `https://purvanshi.github.io/documents/Multimodalppt.pdf`, when I saw the code I could not find where it is.

Thank you very much and I am looking forward to your reply.

Best wishes

**Comment by Justin1904** [comment]
This repo implements [this specific paper](https://arxiv.org/abs/1707.07250). I don't see how it relates to the two links you shared, except that the broader topic is all on multimodality. If you are looking for where the different modalities are fused together, it is [here](https://github.com/Justin1904/TensorFusionNetworks/blob/ef0e78b5583159de9b74ef2cdef6031bd9f94b37/model.py#L146-L160). For the loss function, the paper I referred to as well as this repo does not propose/implement any novel loss function and uses a conventional L1 loss for regression [here.](https://github.com/Justin1904/TensorFusionNetworks/blob/ef0e78b5583159de9b74ef2cdef6031bd9f94b37/train.py#L87)
